api-mock/redirect-middleware.js

```diff
@@ -0,0 +1,11 @@
+module.exports = function(req, res, next) {
+  if (req.params['0'] === '/timeseries') {
```

Comment:
I'd change `req.params['0']` into `req.path` in this file. Other than that, it all looks nice

Reply:
Changed req.params['0'] into req.path